apps/fluent-tester/src/FluentTester/FluentTester.tsx

```diff
@@ -12,10 +12,8 @@ import { useTheme } from '@fluentui-react-native/theme-types';
 import { ThemePickers } from './theme/ThemePickers';
 
 // uncomment the below lines to enable message spy
-/*
```

Comment:
Is this intended to be committed?

Reply:
glossed over it reviewing my changes but tbh I'm somewhat tempted to leave it in -- I'm always uncommenting it for debug purposes and keeping it there as a local change is mildly frustrating.. but there are other ways to accommodate that I should look into -- reverting